### PR TITLE
Add process to send email verification

### PR DESCRIPTION
**Pull request by AngeloCG97**
## Lifebank on-boarding flow
#215

### What does this PR do?

- Create html to email verification.
- Create a new table to management code verification.
- Edit pregister lifebank, add new column "email_verified" and delete column verificarion_code
- Modify process to valide email, delete duplicate funtion.

### How should this be tested?
- Run make run
- go to localhost:3000

### CODE REVIEW

File: hapi/src/api/pre-register.api.js [hunk 1]
```diff
@@ -14,14 +14,28 @@ const INSERT = `
       coordinates
       immunity_test
       invitation_code
+    }
+  }
+`
+
+const INSERTVERIFICATEEMAIL = `
```
Comment (JustinCast) [inline hapi/src/api/pre-register.api.js:21]: Ops, we need to correct this typo

File: hapi/src/api/pre-register.api.js [hunk 2]
```diff
@@ -14,14 +14,28 @@ const INSERT = `
...
+  mutation ($verificate_email: verificate_email_insert_input!) {
```
Comment (JustinCast) [inline hapi/src/api/pre-register.api.js:22]: `verification_email` ?

Comment (JustinCast) [reply hapi/src/api/pre-register.api.js:22]: `verificate_email_insert_input` ?

File: hapi/src/api/pre-register.api.js [hunk 3]
```diff
@@ -14,14 +14,28 @@ const INSERT = `
...
+      email
       verification_code
     }
   }
 `
-const insert = (preregister_lifebank) => {
-  return hasuraUtils.request(INSERT, { preregister_lifebank })
+
+const insertLifebank = (preregister_lifebank) => {
+  return hasuraUtils.request(INSERTLIFEBANK, { preregister_lifebank })
```
Comment (JustinCast) [inline hapi/src/api/pre-register.api.js:31]: Check these typos, please

**Review by JustinCast — CHANGES_REQUESTED** [review]
Please check your spelling

**Review by JustinCast — APPROVED** [review]

**Review by xavier506 — APPROVED** [review]
@AngeloCG97  please see code fctor comment

**Pull request merged**